imports/actions/auth.js

```diff
@@ -0,0 +1,160 @@
+import { Meteor } from 'meteor/meteor';
+import Accounts from 'meteor-apollo-accounts-client';
+import Router from 'react-router-redux';
+
+const SUCCESS_FLASH_TIMEOUT = 2500;
+const internals = {};
+
+internals.flashSuccessThenInit = (dispatch, key, successMethod, successVal) => {
+  dispatch(successMethod(successVal));
+  setTimeout(() => {
+    dispatch(internals.setInitVal(key));
+  }, SUCCESS_FLASH_TIMEOUT);
+};
```

Comment:
'Flashing' success can be used as a trigger to show a success message for a short period of time, then return back to an init state